test/runtests.jl

```diff
@@ -1,3 +1,5 @@
+include("toml_consistency.jl")
```

Comment:
Let's move all the old tests into a new folder: `test/old/`

Reply:
Done